RefRed/calculations/compare_two_lrdata.py

```diff
@@ -1,4 +1,4 @@
-import RefRed.constants
+THETA_TOLERANCE = 0.05
```

Comment:
Should this be stored under one of the modules in `config/`?

Reply:
I think that putting parameters in a python file isn't very clean. I think that if parameters are coded in a python file, they should be with the code that uses it. If they are real parameters that can be changes, then they should be in a loadable json file (or similar editable settings file). Let's chat about it.

Reply:
OK I think I've been convinced by someone else about a better approach :)

Reply:
a config file?